### ISSUE TITLE
What are the hyper-parameters used to training the Carvana dataset?

### ISSUE THREAD

**Issue by dktemp888** [milesial/Pytorch-UNet#137]
Hi @milesial, I would like to reproduce the training on Carvana dataset before jumping to my own dataset. With the default training parameters, it seems like it doesn't converge at least from first few epochs. see training loss below:
<img width="351" alt="Screen Shot 2020-03-03 at 9 41 42 PM" src="https://user-images.githubusercontent.com/61772475/75848945-7bb39400-5d98-11ea-8e80-7e8d76d8db9a.png">
dice/test:
<img width="351" alt="dice" src="https://user-images.githubusercontent.com/61772475/75850702-bfa89800-5d9c-11ea-9876-eee36a259fca.png">

The parameters I use:
epochs: 5
batch: 1 or 8
-l: 0.1 or 0.001
I also tried Adam in addition to RMSprop. All give me similar training loss result like above.

In addition: check the above figure - is it normal that the training loss up and down even within one epoch?

**Comment by leijue222** [comment]
With 1700 images and 1700 target masks.
Set epoch to 100, the other parameters are default same as miles remdeme.md.
Until the epoch is almost equal to 50，the trained model has a little effect, not a black image.

**Comment by dktemp888** [comment]
Thanks @leijue222 for the comments. It turned out when i switch to GPU training everything works in terms of training loss and validation result! Still don't know why with CPU training. Can anyone shade a light on it?

From GPU to CPU the only change i made is to set pin_memory to False when call dataLoader.
